grc20-core/src/mapping/entity/utils.rs

```diff
@@ -163,6 +178,90 @@ impl EntityRelationFilter {
     }
 }
 
+/// Filter used to:
+/// - Traverse to inbound or outbound relation
+#[derive(Clone, Debug, Default)]
+pub struct TraverseRelationFilter {
```

Comment:
Rename to `TraverseRelationSubquery` since it's not strictly a "filter", but also changes the nature of the returned data.

In general, subqueries that end in "Filter" (e.g.: "TypeFilter") imply that they do not change the nature of the returned data.